### PR TITLE
fix: add a way to package media themes as web comp

### PR DESCRIPTION
**Pull request by luwes**
Similar to @heff 's comment https://github.com/muxinc/media-chrome/pull/362#discussion_r1034168845

I think maybe an extends makes sense here with a static `template` property.

### CODE REVIEW

File: src/js/media-theme-element.js [hunk 1]
```diff
@@ -42,25 +46,30 @@ export class MediaThemeElement extends window.HTMLElement {
...
-        this.createRenderer();
-      }
+          if (directive) {
+            t.setAttribute('directive', directive);
+            t.setAttribute('expression', t.getAttribute(directive));
+          }
+        });
+
+      this.createRenderer();
+      this.#prevTemplateHTML = this.template.innerHTML;
```
[the remark this answers is not in the document]
Comment (luwes) [reply src/js/media-theme-element.js:72]: this *is* the string representation, no node references are kept here. it's a pure string

(template.textContent || template.innerText would not include the html tags in string)

[the remark this answers is not in the document]
Comment (cjpillsbury) [reply src/js/media-theme-element.js:72]: d'oh/d'uh.

**Review by cjpillsbury — APPROVED** [review]
🚢 🇮🇹

**Review by gkatsev — APPROVED** [review]

**Review by heff — APPROVED** [review]
I like it 👍

**Pull request merged**